README.md

```diff
@@ -28,12 +36,15 @@ to the generated `coverage.xml`
   
         $ covertool -cover all.coverdata -output coverage.xml -src src/
 
-## Rebar
+### Rebar
 
 Configure rebar to generate reports in `Cobertura` format:
 
 ```
-{plugins, [rebar_covertool]}.
+{deps, [
+    {covertool, "", {git, "https://github.com/idubrov/covertool.git", {branch, master}}}
```

Comment:
Shouldn't this be updated to `https://github.com/covertool/covertool.git`?

Reply:
Thanks for the catch, this was some copy-pasta. Should be addressed now.